nb-extension/js/widgets/DeclWidgetModel.js

```diff
@@ -1,46 +1,41 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
 
-define(function() {
+define(["nbextensions/widgets/widgets/js/widget"], function(Widget) {
```

Comment:
Is this a fully supported path, or just something that happens to work? That is, will the Notebook continue to provide the base WidgetModel at this path or will it change in the near future? If the latter, can we do that now instead?

cc @parente